README.md

```diff
@@ -2,16 +2,16 @@
 
 ## Installation
 
-Install extension via `composer` and add `typo3conf/AdditionalConfiguration.php` (or use symlinks):
+Install extension via `composer` and add `config/system/additional.php` (or use symlinks):
 
 ```
 <?php
-defined('TYPO3_MODE') || exit('Access denied.');
+defined('TYPO3') || die('Access denied.');
 
-require_once __DIR__ . '/ext/cron_context/Default.php';
+require_once \TYPO3\CMS\Core\Core\Environment::getProjectPath() . '/vendor/cron-eu/cron-context/Default.php';
```

Comment:
Leider habe ich keinen anderen Weg gefunden den Pfad anzugeben, die externen EXT sind nicht mehr unter `private/typo3conf/ext/` sondern unter `vendor/`.
Aufrufe wie `\TYPO3\CMS\Core\Utility\ExtensionManagementUtility::extPath()` funktionieren nicht in der `additional.php` da der PackageManager noch nicht existiert.

Reply:
Im CI php build funktioniert dieser Pfad nicht https://git.cron.eu/fvvag/fvvag-base/-/jobs/136049#L200

Reply:
Es lag an den "runner caches", die habe ich in der "Pipeline" screen gelöscht und dann lief das durch. Da wird `vendor/` gecached, vermutlich war da was kaputt.

Reply:
Siehe https://git.cron.eu/fvvag/fvvag-base/-/jobs/136087